pkgs/native_assets_cli/lib/src/code_assets/config.dart

```diff
@@ -346,6 +361,46 @@ extension CodeAssetLinkOutput on LinkOutputAssets {
       .toList();
 }
 
+/// Access to the C API for this version of Dart.
+///
+/// Only `dart_api_dl.h` should be used. `dart_api_dl.c` shoud be compiled into
+/// your `CodeAsset`.
+///
+/// Note: If you're precompiling code assets, you must check that [version] is
```

Comment:
This documentation might not be sufficient in the long wrong - how would I check that?

Reply:
One should check the major version is identical and the minor version is at least what you need.